cg/cli/workflow/balsamic/options.py

```diff
@@ -5,6 +5,14 @@
 OPTION_DRY = click.option(
     "-d", "--dry-run", help="Print command to console without executing", is_flag=True
 )
+OPTION_GENOME_VERSION = click.option(
+    "-g",
+    "--genome-version",
+    show_default=True,
+    default="hg19",
+    type=click.Choice(["hg19", "hg38", "canfam3"]),
```

Comment:
It would be great if these were defined in constants and imported here instead of hard coded. Would be great with as few genome build strings as possible to keep everything consistent and easy to change in the future

Reply:
I agree: [f23a1c1](https://github.com/Clinical-Genomics/cg/pull/1477/commits/f23a1c1b738879dc0ab94f40b5539724c2c7615e)